.github/workflows/pull-request.yaml

```
@@ -54,35 +54,49 @@ jobs:
-            echo "Detected workspace changes: $changed"
-            echo "changed-workspaces=${changed}" >> "$GITHUB_OUTPUT"
+            # Create JSON array manually to ensure single line output
+            json_array="["
+            for i in "${!changed_workspaces[@]}"; do
+              if [ $i -gt 0 ]; then
+                json_array+=","
+              fi
+              json_array+="\"${changed_workspaces[i]}\""
+            done
+            json_array+="]"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)